metadata/labels.go

```diff
@@ -41,6 +44,9 @@ var (
 	// AutomatedLabel is the label name for marking a Release as automated
 	AutomatedLabel = fmt.Sprintf("release.%s/automated", rhtapDomain)
 
+	// ServiceNameLabel is the label used to specify the service associated with an object
+	ServiceNameLabel = fmt.Sprintf("%s/%s", rhtapDomain, "service")
```

Comment:
This would be `appstudio.openshift.io/service` how does that help us?

Reply:
It would (from what I understood) add the label as `appstudio.openshift.io/service: release` to Pipelines created by the operator (adapter lines 494, 534 and 576), so the cache would only consider those ones. I did not put much brain on the label name as I basically copied this from another PR as this looked enough to "categorize" what should be cached.

Reply:
Maybe you meant `s/"service"/ServiceName/` then? You are just combining rhtapDomain variable with the string "service" right now

Reply:
If the suggested replacement is made, the label would look like in the end `appstudio.openshift.io/service: release`, as what adds it to the PipelineRun is:
```
WithLabels(map[string]string{
...
metadata.ServiceNameLabel: metadata.ServiceName,
}).
```
so it adds the label as expected, correct? I understood that the  ServiceNameLabel" is the "key" and the ServiceName (`release`) is the value - and the WithLabel makes it `appstudio.openshift.io/service: release`

https://github.com/konflux-ci/release-service/pull/655/files#diff-6f40aa8049566b39f0c38b10d839c98d6b9298c98d783c4b82d8e6ba368067d4R494

Reply:
`fmt.Sprintf("%s/%s", rhtapDomain, "service")` you are combining the variable `rhtapDomain` with the *string* "service" look at the quotes. Nothing is adding `release` to the label

Reply:
Gotcha. I thought your intent was to have `release` in the label name, not just the value